core/src/main/java/org/apache/carbondata/core/cache/dictionary/AbstractDictionaryCache.java

```diff
@@ -59,6 +59,10 @@ public AbstractDictionaryCache(CarbonLRUCache carbonLRUCache) {
     initThreadPoolSize();
   }
 
+  @Override public void put(DictionaryColumnUniqueIdentifier key, Dictionary value) {
```

Comment:
move @Override to previous line
please follow this in future

Reply:
ok